crates/tx5-connection/src/framed.rs

```diff
@@ -157,6 +159,24 @@ impl FramedConn {
         &self.pub_key
     }
 
+    /// Returns `true` if we sucessfully connected over webrtc.
+    pub fn is_using_webrtc(&self) -> bool {
+        if let Some(conn) = self.weak_conn.upgrade() {
+            conn.is_using_webrtc()
+        } else {
+            false
+        }
+    }
+
+    /// Get connection statistics.
+    pub fn get_stats(&self) -> ConnStats {
+        if let Some(conn) = self.weak_conn.upgrade() {
+            conn.get_stats()
+        } else {
+            ConnStats::default()
```

Comment:
Return `Option::None` if there are no stats around rather than default?

Reply:
Yeah, on the startup side, i think it makes sense for the stats to be zeroed until we start collecting data. On the shutdown side it's a little weirder for them to jump back to zero, but it makes dealing with the api easier. I vote for keeping it direct for now and we can see if the usage gets confusing.